docs/htdocs/info/docs/tools/vep/script/vep_download.html

```diff
@@ -116,15 +116,18 @@ <h2 id="new">What's new?</h2>
     <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_RELEASE_DATE]])</i></h4>
 
   <ul>
-    <li>New plugins for command line use: </li>
+    <li>New plugin (supported on CLI, REST, and web): </li>
       <ul>
-        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/IntAct.pm" rel="external">IntAct</a> - annotates human variants which fall in interaction sites, as described in the IntAct database</li>
-        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/CAPICE.pm" rel="external">CAPICE</a> - integrates scored from a machine-learning-based method for prioritizing pathogenic variants (GRCh37 only)</li>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/EVE.pm" rel="external">EVE</a> - annotates human variants using EVA classification method based solely on evolutionary sequences (GRCh38 only)</li>
       </ul>
-    <li>Nextflow pipeline:</li>
+    <li>Plugins now available in REST and web: </li>
       <ul>
-        <li>A new configurable pipeline is available to run Ensembl VEP efficiently on large scale VCF</li>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/GO.pm" rel="external">GO</a> - retrieves Gene Ontology terms associated with transcripts/translations via the Ensembl API</li>
```

Comment:
The GO plugin doesn't get data `via the Ensembl API` anymore (unless the user explicitly asks for it). In [VEP_plugins#494](https://github.com/Ensembl/VEP_plugins/pull/494/files#diff-8dda975ddb3d9305c876e6745ca5177af8dca519d080dd6e364f51a4d60e9685), I changed the description to rectify this:

```suggestion
        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/GO.pm" rel="external">GO</a> - retrieves Gene Ontology terms associated with transcripts/translations</li>
```